themes/startwords/layouts/index.html

```diff
@@ -14,19 +14,18 @@ <h1 class="sr-only">{{ .Site.Title }}</h1>
     </div>
     <div class="issue-summary inverted grid">
         <div class="container">
-            {{ range first 1 (where .Site.AllPages "Type" "issue") }}
-            <span class="number">Issue {{ .Params.number }}</span>
-            <h2 class="theme">{{ .Params.theme }}</h2>
+            {{ $issue := index (where .Site.AllPages "Type" "issue") 0 }}
+            <span class="number">Issue {{ $issue.Params.number }}</span>
+            <h2 class="theme">{{ $issue.Params.theme }}</h2>
             <p class="summary">
                 {{ .Summary | plainify }}
```

Comment:
change this to `{{ $issue.Summary | plainify }}` to bring back the issue summary.